elabapy/Manager.py

```diff
@@ -27,6 +27,8 @@ def create_template(self):
     def get_all_experiments(self, params={'limit':15,'offset':0}):
         """
             This function returns a list of all experiments.
+            params is a dict with keys 'limit' and/or 'offset'
+            param_name has to be 'params' to be in accordance with the current requests-doku (https://2.python-requests.org/en/master/api/#requests.get)
```

Comment:
use: "current requests documentation"

Reply:
I've added your suggestion within the last commit.